### ISSUE TITLE
[Screen Reader-Command Palette]-Screen reader user will not be able to identify the purpose of New Tab control if role and state are not appropriate.

### ISSUE THREAD

**Issue by DasaradhaKamjula** [microsoft/terminal#7908]
**User Experience:**
Users who rely on screen reader will not be able to get the info of sub menu if role/state is not provided properly

**Test Environment:**
App: Windows Terminal Preview
Feature: Command Palette
Screen Reader: Narrator
OS: Win 10 2004 Build (19041.508)

**Repro Steps:**
1.Open Windows terminal Preview app
2.Open Command Palette using shortcut (Ctrl+shift+p)
3.Search for New tab command in the search field
4.Navigate to New Tab list item having sub menu  and Observe whether screen reader is providing proper info or not

**Actual:**
Screen reader user will not be able to identify the purpose of New Tab control if role and state are not appropriate. Screen reader announces the name and position only and also Multiple New tab list items are present in the list and the announcement is same which will confuse user.

Note: Same issue repro for Split pane and Set color scheme controls having Sub menu

**Expected:**
State should announce state as Collapsed or it should provide extra role description like "Has pop Up" which helps in differentiating the New tab list items and also the purpose of New tab having sub menu.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by DHowett** [comment]
This fix will be deployed to the Stable channel tomorrow (Thursday).

**Comment by zadjii-msft** [comment]
@Chandresh1007, @DasaradhaKamjula, @Priyanka-Chauhan123 - can you confirm that this is fixed, and close it if it is? This should have shipped in 1.6. Thanks!

**Comment by Priyanka-Chauhan123** [comment]
Issue is fixed. Verified on below environment.
Windows Terminal App version: 1.9.1942.0.
OS: Windows 10 build 21390.2025.
Hence, closing the bug.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as issue has been fixed.
